src/fbs/header.fbs

```diff
@@ -44,6 +44,14 @@ enum ColumnType: ubyte {
 table Column {
   name: string (required);      // Column name
   type: ColumnType;             // Column type
+  title: string;                // Column title
+  description: string;          // Column description (intended for free form long text)
+  width: int = -1;              // Column values expected width (-1 = unknown) (currently only used to indicate the number of characters in strings)
+  precision: int = -1;          // Column values expected precision (-1 = unknown) as defined by SQL
+  scale: int = -1;              // Column values expected scale (-1 = unknown) as defined by SQL
+  nullable: bool = true;        // Column values expected nullability
+  unique: bool = false;         // Column values expected uniqueness
+  primary_key: bool = false;    // Indicates this column have been (part) of primary key
```

Comment:
```suggestion
  primary_key: bool = false;    // Indicates this column has been (part of) a primary key
```
So the intent is that if several columns are flagged primary_key = true, this is a multi-column primary key ?

Reply:
Yes